PowerShellGet/PSModule.psm1

```diff
@@ -7199,7 +7548,8 @@ function New-PSGetItemInfo
 
         $PSGetItemInfo = Microsoft.PowerShell.Utility\New-Object PSCustomObject -Property ([ordered]@{
                 Name = $swid.Name
-                Version = [Version]$swid.Version
+                Version = $swid.Version #-split '-',2 | Select-Object -First 1
+                #Prerelease = $swid.Version -split '-',2 | Select-Object -Skip 1

```

Comment:
Yes, I plan to, will do that once all bugs are fixed.  I haven't quite determined yet if prerelease needs to be pulled into it's own field or not.

---
In reply to: [132598640](https://github.com/rebro-msft/PowerShellGet/pull/1#discussion_r132598640) [](ancestors = 132598640)